src/pages/SearchPage.js

```diff
@@ -0,0 +1,175 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import Header from '../components/Header';
+import Poster from '../components/Poster';
+import Navigation from '../components/Navigation';
+import ScreenReaderOnly from '../components/ScreenReaderOnly';
+import LoadingSpinner from '../components/LoadingSpinner';
+
+const {
+  REACT_APP_API_BASE_URL_SEARCH,
+  REACT_APP_API_KEY,
+  REACT_APP_API_LANGUAGE,
+} = process.env;
+
+export default function SearchPage() {
+  const [results, setResults] = useState([]);
+  let [isLoading, setIsLoading] = useState(false);
+  let searchUrl = '';
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    async function loadData() {
+      setIsLoading(true);
+      try {
+        const form = event.target;
+        const query = form.elements.search.value;
+        searchUrl = `${REACT_APP_API_BASE_URL_SEARCH}?api_key=${REACT_APP_API_KEY}&query=${query}&language=${REACT_APP_API_LANGUAGE})`;
+        const response = await fetch(searchUrl);
+        const data = await response.json();
+        setResults(data.results);
+        form.reset();
+      } catch (error) {
+        console.error(error);
+      }
+      setIsLoading(false);
+    }
+    loadData();
+  }
```

Comment:
If you have no request limits for your API, maybe it would be nice to use an `onChange` event here instead of `onSubmit`. So your users can see the search results already while typing :)